docs/config/index.md

```diff
@@ -241,6 +241,11 @@ export default defineConfig(({ command, mode }) => {
 
   Vite has a list of "allowed conditions" and will match the first condition that is in the allowed list. The default allowed conditions are: `import`, `module`, `browser`, `default`, and `production/development` based on current mode. The `resolve.conditions` config option allows specifying additional allowed conditions.
 
+  :::warning the property ending with "/" in "exports" is not resolved
+  Property ending with "/" is not supported. Because it is deprecated in Node.js. In [this spec](https://nodejs.org/api/packages.html#package-entry-points), `*` subpath patterns should be prefered.
+  Please modify the `exports` field by directly changing the source code, or using [patch-package](https://www.npmjs.com/package/patch-package)
+  :::
```

Comment:
I think we can simplify the text a bit since this is an edge case of a deprecated API, maybe something like:

```
:::warning Subpath exports
In some cases, exports ending with `/` may not work as it's [deprecated API](https://nodejs.org/api/packages.html#package-entry-points). Please contact to package author to use `*` subpath patterns instead.
:::
```

Feel free to modify a bit where you see fit! I also mentioned notifying package authors as that would fix the source directly, and we had a similar error message like so too.

Reply:
Also because I've seen cases where packages using `/` indeed does work, but I think in that specific issue it hit an edge case that it doesn't resolve the `index.js`.

Reply:
I simplify the text. the text is:
resolve subpath export
---
Property ending with "/" is not supported. It is deprecated. Please contact to package author to use [* subpath patterns](https://nodejs.org/api/packages.html#package-entry-points) instread.